packages/public/frontend/src/assets.ts

```diff
@@ -1,4 +1,8 @@
-const assetPrefix = process.env.ASSET_PREFIX || ''
+let assetPrefix = process.env.ASSET_PREFIX || ''
+
+export function setAssetPrefix(path) {
```

Comment:
Why is this needed? Shouldn't this always be `process.env.ASSET_PREFIX`?

Reply:
Unfortunately not when this code is run on the client, so we need to set it explicitly

Reply:
I see. Then why is this different compared to the other usages of the asset prefix? So far, we pass those via props, don't we?

Furthermore: it seems like there are already a couple usages of the assetPrefix in the project. What do you think about an `assetPrefixContext` that we pretty much set once at the root? And every component that needs it (doesn't matter how deep in the tree), uses that context.

Reply:
I haven't decided yet which way to go, a context would be great and maybe it can we implemented at nextjs level, so pages don't need to care about it ... I feel this would fit into a separate PR

Reply:
That's fine by me. But the current usages should be done in a similar way, right?

Reply:
done